sdk/src/sealance/merkletree-support.ts

```diff
@@ -0,0 +1,237 @@
+import { Field, Plaintext, Poseidon4 } from "../wasm.js";
```

Comment:
This should be imported directly from the SDK import.

```suggestion
import { Field, Plaintext, Poseidon4 } from "@provablehq/sdk/testnet.js" to ensure the `wasm` loads correctly;
```